docs/pages/api-docs/data-grid/data-grid.md

```diff
@@ -66,7 +66,7 @@ import { DataGrid } from '@material-ui/data-grid';
 | <span class="prop-name">onColumnHeaderLeave</span> | <span class="prop-type">(param: GridColumnHeaderParams, event: React.MouseEvent) => void</span> |   | Callback fired when a mouse leave event comes from a column header element. |
 | <span class="prop-name">onColumnOrderChange</span> | <span class="prop-type">(param: GridColumnOrderChangeParams, event: React.MouseEvent) => void</span> |   | Callback fired when a column is reordered. |
 | <span class="prop-name">onColumnResize</span> | <span class="prop-type">(param: GridColumnResizeParams) => void</span> |   | Callback fired when a column is resizing. |
-| <span class="prop-name">onColumnResizeCommitted</span> | <span class="prop-type">(param: GridColumnResizeParams) => void</span> |   | Callback fired when a column is resized. |
+| <span class="prop-name">onColumnWidthChange</span> | <span class="prop-type">(param: GridColumnResizeParams) => void</span> |   | Callback fired when a column is resized. |
```

Comment:
I feel like the description of `onColumnResize` and `onColumnWidthChange` could be clearer.
To better indicate that `onColumnResize` is triggered multiple times during the resize process and `onColumnWidthChange` only at the end.

Reply:
Updated.

Reply:
@m4theushw We probably want to update the markdown too?

Reply:
@oliviertassinari Yeah, I missed the markdown. Now it's fixed. I also updated https://deploy-preview-1967--material-ui-x.netlify.app/components/data-grid/columns/#resizing to explain better these two callbacks.